build/debs/bionic/kubeadm/debian/rules

```diff
@@ -2,15 +2,20 @@
 # -*- makefile -*-
 
 #export DH_VERBOSE=1
+LOCAL_OUTPUT_DIRECTORY=$(shell go env GOPATH)/src/k8s.io/kubernetes/_output/dockerized/bin/linux
```

Comment:
golang claims to deprecate GOPATH in 1.13, and i can't find information on their deprecation policy but seems it might be removed in 1.15?
wondering whether `go env GOPATH` will still work after that.

Reply:
Ack. will switch to `GOROOT` (like we do in https://github.com/kubernetes/kubernetes/blob/master/hack/lib/golang.sh#L517)